test/unit/uk/gov/hmrc/organisationsdetailsapi/services/ScopesHelperSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.organisationsdetailsapi.services
+
+import org.scalatest.BeforeAndAfterEach
+import play.api.libs.json.Json
+import uk.gov.hmrc.organisationsdetailsapi.config.ApiConfig
+import uk.gov.hmrc.organisationsdetailsapi.services.{ScopesHelper, ScopesService}
+import utils.TestSupport
+
+class ScopesHelperSpec
+  extends TestSupport
+    with ScopesConfig
+    with BeforeAndAfterEach {
+
+  "Scopes helper" should {
+
+    val scopesService = new ScopesService(mockConfig) {
+      override lazy val apiConfig: ApiConfig = mockApiConfig
+    }
+
+    val scopesHelper = new ScopesHelper(scopesService)
+
+    "return replaced employerRef" in {
+      val employerRef = "247%2FZT6767895A"
+      val result =
+        scopesHelper.getQueryStringWithParameterisedFilters(List(mockScope8), mockEndpoint4, employerRef)
+      result shouldBe s"employer(employerAddress(line1,line2,line3),employerDistrictNumber,employerName,employerSchemeReference),payments&filter=employerRef eq '$employerRef'"
+    }
+
+    "return correct query string" in {
+      val result =
+        scopesHelper.getQueryStringFor(List(mockScope2), mockEndpoint1)
+      result shouldBe "employer(employerAddress(line1,line2,line3),employerDistrictNumber,employerName,employerSchemeReference),payments"
```

Comment:
Test are all looking for employments which feels wrong. I would rather they were not here for the time being.

Reply:
If we had to keep simple tests I would rather they were cut down tests returning simple objects than be employments specific. 

I.e. fields(one, two, three) 

If they are only proving the handler pattern only this should be possible.